test/scrabble_test.rb

```diff
@@ -5,8 +5,42 @@
 
 class ScrabbleTest < Minitest::Test
```

Comment:
Overall, your test suite looks good, though I would have liked to see a test for your helper method `multiply_string`